lib/assets/javascripts/turbograft/turbohead.coffee

```diff
@@ -26,8 +26,7 @@ updateScriptTags = (activeDocument, newScripts, callback) ->
   )
 
 extractTrackedAssets = (doc) ->
-  for node in doc.head.children when node.dataset.turbolinksTrack?
-    node
+  [].slice.call(doc.querySelectorAll('[data-turbolinks-track]'))
```

Comment:
How about `Array.prototype.slice` instead of `[].slice`? http://stackoverflow.com/questions/9006553/slice-or-array-prototype-slice suggests prototype is more performant, unfortunately jsperf is dead :(